### DISCUSSION TITLE
GET / - Uncaught TypeError: Saloon\Http\Senders\GuzzleSender::createResponse(): Argument #2

### DISCUSSION THREAD

**Discussion by AerKov** [saloonphp/saloon#261] [Q&A] [answered]
Good afternoon. When I try to run this code it throws an error:
GET / - Uncaught TypeError: Saloon\Http\Senders\GuzzleSender::createResponse(): Argument #2 ($guzzleResponse) must be of type Psr\Http\Message\ResponseInterface, null given, called in ....
Tell me, please, what could be the problem?

```php
<?php
//CheckRequest.php
use Saloon\Http\SoloRequest;
use Saloon\Enums\Method;

class CheckRequest extends SoloRequest
{
    protected Method $method = Method::GET;
    
    public function resolveEndpoint(): string
    {
        return 'https://pokeapi.co/api/v2/pokemon';
    }

}
``` 
```php
<?php 
// index.php
    require './vendor/autoload.php';
    require './check/CheckRequest.php';

    $request = new CheckRequest;
    $response = $request->send();
    
echo '<pre>';
    print_r($response);
echo '</pre>';
```

**Comment by Sammyjo20** [comment]
Hey @AerKov!

What version of Saloon are you currently using? This was a previous issue but was fixed in v2.6.4 and also had a related issue here:

https://github.com/Sammyjo20/Saloon/issues/188

How does Saloon’s version look in your composer.json file?

**Reply by AerKov** [reply, marked as answer]
Thanks a lot. Didn't notice right away. It worked!!! Changed the version to 2.6.4 in composer.json and made the `composer update`
